platform/android/os_android.cpp

```diff
@@ -444,25 +444,27 @@ void OS_Android::process_touch(int p_what, int p_pointer, const Vector<TouchPos>
 				}
 				touch.clear();
 			}
-
 		} break;
-		case 3: { // add tuchi
-
-			ERR_FAIL_INDEX(p_pointer, p_points.size());
```

Comment:
`p_pointer` can be any integer regardless `p_points.size()`
`p_points` array can be like this.
```
[ id:0, x:10, y:10 ],
[ id:2, x:20, y:20 ],
[ id:3, x:30, y:30 ]
```
and `p_pointer` indicates last id which can be `3`